tests/visuals/test_visual_app.py

```diff
@@ -21,7 +21,12 @@
 import pytest
 
 from rectools import Columns, ExternalId
-from rectools.visuals.visual_app import AppDataStorage, ItemToItemVisualApp, StorageFiles, TablesDict, VisualApp
+from rectools.visuals.visual_app import (AppDataStorage,
```

Comment:
Please run `make format` command in your terminal before pushing your changes. It will run `isort` and `black` to fix code style.
Also it is useful to run `make lint` command to check that all linters are passing locally. Right now they are failing because of unusual formatting.